docs/modules/getting_started/examples/code/trino.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: simple-trino
 spec:
   image:
-    productVersion: 396
+    productVersion: "396"
```

Comment:
We could go for the "latest" version which is "403-stackable0.1.0"

Reply:
not here